client/src/pages/CreateUserProfile.js

```diff
@@ -0,0 +1,219 @@
+               <span>New York,</span>
+               <span>NY,</span>
+               <span>USA</span>
+            </Menu.Item>
+            <Menu.Divider/>
+        </StyledMenu>
+    );
+    return <Dropdown overlay={menu}>
+        {children}
+    </Dropdown>
+}
+
+const InputGroup = styled.div`
+    --py: ${theme.typography.size.xxsmall};
+    --my: ${theme.typography.size.xxxlarge};
+    
+    margin: var(--my) 0 var(--my) 0;
+
+    h3 {
+        color: ${theme.colors.primary};
+        font-weight: bold;
+    }
+    input {
+        color: ${theme.colors.darkerGray};
+        font-size: ${theme.typography.size.xlarge};
+        width: 100%;
+        margin-bottom: var(--my);
+        padding: var(--py) 0 var(--py) 0;
+
+    }
+    label {
+        font-family: ${theme.typography.font.family.button};
+        font-size: ${theme.typography.size.large};
+        font-weight: 500;
+    }
+`
+
+const UnderlineLink = styled.p`
+    --color: ${theme.colors.darkGray};
+    color: var(--color) !important;
+    a {
+        color: var(--color);
+        text-decoration: underline;
+    }
+    font-size: ${theme.typography.size.small};
+`;
+
+const CreateProfile = () => {
+return <Container>
+    <Flex className="image-container" direction="column">
+        <img src={PersonalDataImage} alt="two people standing next to a list of personal items" />
+    </Flex>
+    <Flex className="form-container" direction="column">
+        <WhiteSpace size="xl"/>
+        <ProfileFormGroup>
+            <CustomH1 color={theme.colors.darkerGray} fontweight="bold">Create your Profile</CustomH1>
+            <InputGroup>
+                <TextInput label="E-mail" />
+                <TextInput label="Name" />
+                <DropdownMenu>
+                    <div id="dropdown-anchor" style={{'position': 'relative'}}>
+                        <TextInput label="Address"/>
+                    </div>
+                </DropdownMenu>
+            </InputGroup>
+            <CheckboxGroup description="I am traveling" />
+            <CheckboxGroup description="Don't show my address" />
+            <InputGroup>
+                <Title>I want to</Title>
+                <CheckboxGroup label="Volunteer" />
+                <CheckboxGroup label="Donate" />
+                <CheckboxGroup label="Share Information" />
+            </InputGroup>
+            <InputGroup>
+                <Title>I need</Title>
+                <CheckboxGroup 
+                label="Medical Help" 
+                description="I have symptoms of COVID-19"/>
+                <CheckboxGroup 
+                label="Other Help" 
+                description="I need assitance getting groceries, medicine, etc." />
+            </InputGroup>
+            <InputGroup>
+                <SubmitButton primary="true">Create Profile</SubmitButton>
+                <CheckboxGroup>
+                    <UnderlineLink>By signing up, I agree to the <a href="#">Terms and Conditions</a>
```

Comment:
The order of the UnderlineLinks are switched, should be Privacy and Policy first. 

You can find the `href` values in `routes.js` for both urls to both pages.